src/software/simulated_tests/simulated_test_fixture.h

```diff
@@ -138,15 +138,16 @@ class SimulatedTestFixture : public ::testing::Test
             non_terminating_function_validators);
 
     /**
-     * Puts the current thread to sleep until the difference between the current wall time
-     * and the wall_start_time is >= the difference between current_time and zero. This
-     * "synchronizes" wall time with simulation time by slowing down execution if needed.
+     * Puts the current thread to sleep such that each simulation step will take
+     * the desired about of real-world "wall" time.
      *
-     * @param wall_start_time The time at which the simulated test started, in wall time
-     * @param current_time The current time in the simulated test
+     * @param wall_start_time The time at which the most recent simulation step started,
+     * in wall time
+     * @param desired_wall_tick_time How long each simulation step should take
+     * in wall-clock time
```

Comment:
Is there somewhere mentioning what "wall-clock" time is and how it differs from "wall" time?

Reply:
They're the same, just both common ways of saying the same thing (as opposed to saying "real-time")